gridsome/app/components/Pager.js

```diff
@@ -49,16 +54,20 @@ export default {
         linkProps.exactActiveClass = props.exactActiveLinkClass
       }
 
+      if (!Array.isArray(linkClass)) {
+        linkClass = [linkClass]
+      }
+
       return h(Link, {
-        class: props.linkClass,
+        class: [props.linkClass, ...linkClass],
```

Comment:
```suggestion
        class: [props.linkClass, linkClass],
```
Vue merges arrays and objects into class names internally.